math-magic/src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Component from './components/Calculator';
```

Comment:
- [ ] Kindly fix the structure of the project, since this is giving some errors while running `npm run start` or `npm run build`

![image](https://github.com/misikir21/mama/assets/93630700/b300351b-887e-425e-a11f-b057180badce)

It may maybe because your `math/magic` project it's inside another folder, you could try to put your files from the `mama` folder inside your math-magic to work.

![image](https://github.com/misikir21/mama/assets/93630700/5f6fb7cf-1cf6-404b-b65a-f3d90d204dcb)